pkg/machine/ignition.go

```diff
@@ -166,24 +168,57 @@ Environment=SYSTEMD_CONF=/etc/systemd/system.conf.d/default-env.conf
 Environment=ENVD_CONF=/etc/environment.d/default-env.conf
 Environment=PROFILE_CONF=/etc/profile.d/default-env.sh
 ExecStart=/usr/bin/bash -c '/usr/bin/test -f ${FWCFGRAW} &&\
-	echo "[Manager]\n#Got from QEMU FW_CFG\nDefaultEnvironment=$(/usr/bin/base64 -d ${FWCFGRAW} | sed -e "s+|+ +g")\n" > ${SYSTEMD_CONF} ||\
-	echo "[Manager]\n#Got nothing from QEMU FW_CFG\n#DefaultEnvironment=\n" > ${SYSTEMD_CONF}'
+        echo "[Manager]\n#Got from QEMU FW_CFG\nDefaultEnvironment=$(/usr/bin/base64 -d ${FWCFGRAW} | sed -e "s+|+ +g")\n" > ${SYSTEMD_CONF} ||\
```

Comment:
What happened here with all the spaces/indention

Reply:
No idea: I just ran `go fmt` on all files.  If that's the only thing stopping this from being approved, it's  easy enough to rewrite those changes out of the commit -- but your comment below makes it sound like that's not the case.